ocs_ci/ocs/ui/base_ui.py

```diff
@@ -402,7 +402,7 @@ def login_ui():
 
         # headless browsers are web browsers without a GUI
         headless = ocsci_config.UI_SELENIUM.get("headless")
-        if headless:
+        if not headless:
```

Comment:
why did you change it?
jenkins slave works without console.

Reply:
change reverted